VMware.CloudFoundation.CertificateManagement.psm1

```diff
@@ -1401,9 +1418,42 @@ Function Install-EsxiCertificate {
                 continue
             }
 
+            if ($vcfVersion -eq "5.2") {
```

Comment:
-ge ?

Reply:
addressed in commit e6a7844965249faebca841b4da521bef3b7b40ba